docs/src/tutorials/exporting.md

```diff
@@ -1,51 +1,176 @@
 ---
 title: "Exporting data"
-description: |
-  Platform.sh aims to be a great host, but we never want to lock you in to our service. Your code and your data belong to you, and you should always be able to download your site's data for local development, backup, or to "take your data elsewhere".
+description: See how to export your code, files and service's data.
 ---
 
-{{% description %}}
+Platform.sh never wants to lock you in to our service.
```

Comment:
I'd just note this tone of voice is one of the things we're looking to try to get rid of in the rework. There's a reason it got flagged in the linting.

Reply:
I thought that in that case it wasn't too condescending and a good way to talk about values and against vendor lock-in.